pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -339,6 +357,23 @@ export class ConfigurationCard extends React.Component {
                 { this.state.serverTime }
             </PrivilegedButton>);
 
+        const domain_tooltip = (this.permission.allowed === false
+            ? cockpit.format(_("The user $0 is not permitted to modify realms"),
+                             this.permission.user ? this.permission.user.name : '')
+            : this.realmd.button_tooltip);
+        const domain_disabled = this.permission.allowed === false || this.realmd.button_disabled;
+
+        const domain_button = (
+            <Privileged allowed={ !domain_tooltip }
+                        tooltipId="system_information_domain_tooltip"
+                        excuse={ domain_tooltip }>
+                <Button id="system_information_domain_button" variant="link"
+                        onClick={ () => this.realmd.clicked() }
+                        isInline isDisabled={ domain_disabled } aria-label="join domain">
```

Comment:
Do we really need the aria-label here? This is just a standard button, and the actual text is so much more correct. I. e. when you are already joined, then "join domain" is wrong, and it's also not translated. If we need it, could it just be `this.realmd.button_text` instead?

Reply:
It was in the original code, so... *shrug*

Reply:
Ah, I didn't see it in the diff. Nevermind then.